src/utils.jl

```diff
@@ -186,13 +186,9 @@ end
 Return rotation matrix from vector `u` to vector `v`.
 """
 function uvrotation(u::Vec{2}, v::Vec{2})
-  convert(DCM, CounterClockwiseAngle(∠(u, v)))
+  Angle2d(∠(u,v))
 end
 
 function uvrotation(u::Vec{3}, v::Vec{3})
-  u⃗ = normalize(u)
-  v⃗ = normalize(v)
-  re = √((1 + u⃗ ⋅ v⃗) / 2)
-  im = (u⃗ × v⃗) / 2re
-  convert(DCM, Quaternion(re, im...))
+  return rotation_between(v, u)
 end
```

Comment:
There is a similar function `Rotations.rotation_between`, but the order of arguments is different.
I kept the current behavior, but `rotation_between(u, v)` seems better for its consistency.

```julia
julia> using Meshes, LinearAlgebra, Pkg

julia> Pkg.status("Meshes")
Status `~/tmp/Project.toml`
  [eacbb407] Meshes v0.28.1

julia> uvrotation(normalize(Vec2(1,2)), normalize(Vec2(2,1))) * Vec2(1,2)  # Correct
2-element Vec2 with indices SOneTo(2):
 2.0
 1.0

julia> uvrotation(normalize(Vec3(1,2,3)), normalize(Vec3(2,1,3))) * Vec3(1,2,3)  # Inconsistent
3-element Vec3 with indices SOneTo(3):
 -0.14285714285714293
  2.7142857142857144
  2.5714285714285716

julia> uvrotation(normalize(Vec3(2,1,3)), normalize(Vec3(1,2,3))) * Vec3(1,2,3)  # Desired behavior
3-element Vec3 with indices SOneTo(3):
 2.0
 1.0
 3.0
```

Reply:
Do you think rotation_between could use the (u, v) order in a new release? I am finding it a bit strange that the order is reversed. What is the rationale?